embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/session/EmbraceBackgroundActivityService.kt

```diff
@@ -42,17 +40,11 @@ internal class EmbraceBackgroundActivityService(
     var backgroundActivity: Session? = null
     private val manualBkgSessionsSent = AtomicInteger(0)
 
-    private var lastSendAttempt: Long
+    private var lastSendAttempt: Long = clock.now()
```

Comment:
Should this be set to 0 or something to indicate that we didn't know if there was even a last attempt?

Reply:
Ah, I see this was just what the code used to do. Never mind - feel free to not make this change!